.github/workflows/main.yml

```diff
@@ -29,22 +29,26 @@ jobs:
         sudo curl -o GPG-PUB-KEY-INTEL-OPENVINO-2021 https://apt.repos.intel.com/openvino/2021/GPG-PUB-KEY-INTEL-OPENVINO-2021
         sudo apt-key add GPG-PUB-KEY-INTEL-OPENVINO-2021
         sudo echo "deb https://apt.repos.intel.com/openvino/2021 all main" | sudo tee - a /etc/apt/sources.list.d/intel-openvino-2021.list
-        sudo apt-get update
-        sudo apt-get install -y --no-install-recommends intel-openvino-dev-ubuntu18-2021.2.200
-        sudo apt-get install -y --no-install-recommends python3-setuptools
+        sudo apt update
+        sudo apt install -y --no-install-recommends intel-openvino-dev-ubuntu18-2021.4.689
+        #sudo apt install -y --no-install-recommends intel-openvino-dev-ubuntu18-2021.2.200
+        sudo apt install -y --no-install-recommends python3-setuptools
+        cd /opt/intel/openvino_2021/deployment_tools/model_optimizer/install_prerequisites
+        sudo ./install_prerequisites.sh
+        python3 -m pip freeze
 
     - name: Clone AutoML
       run: |
-        git clone https://github.com/google/automl --depth 1
+        git clone https://github.com/google/automl
+        cd automl
+        #git checkout 96e1fee        
         python3 -m pip install --upgrade pip
         python3 -m pip install numpy
-        cd automl
-        git log
+        git log --pretty=oneline | head -25
         python3 -m pip install -r efficientdet/requirements.txt
-
-    - name: Clone OpenVINO
-      run: |
-        git clone https://github.com/openvinotoolkit/openvino --depth 1
+        #python3 -m pip install tensorflow==2.4.3
+        python3 -m pip install tensorflow-model-optimization
+        python3 -m pip freeze
```

Comment:
Already installed above

Reply:
1. Not sure what you're referring to. This was partially in the original code (closing OpenVino from github master, I removed this whole part and now we depend on the one installed from apt repo). 

2. Removed `tensorflow==2.4.3` (the commented out line - we now rely on the one that gets installed by OpenVino's requirements.txt which gets installed by ./install_prerequisites.sh)

3. `tensorflow-model-optimization` is not installed by anything, but is needed for freezing graph. 

4. `pip freeze` at the end allows us to keep logs of which python packages were used to build the IR.